File: backend/core/entities/participant.go
package entities

type Participant struct {
	ID           string `json:"id"            bson:"_id,omitempty"`
	Name         string `json:"name"          bson:"name"`
	IsNominated  bool   `json:"is_nominated"  bson:"isNominated"`
	IsEliminated bool   `json:"is_eliminated" bson:"isEliminated"`
	ImageUrl     string `json:"image_url"     bson:"imageUrl"`
}

File: frontend/src/models/Participant.ts
export default class Participant {
  id: string;
  name: string;
  imageUrl: string;
  isEliminated: boolean;
  isNominated: boolean;

  constructor(data: any) {
    this.id = data.id;
    this.name = data.name;
    this.imageUrl = data.image_url;
    this.isEliminated = data.is_eliminated;
    this.isNominated = data.is_nominated;
  }
}
